example-library/src/main/java/uk/gov/gchq/palisade/example/config/ExamplePolicyConfiguration.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.example.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import uk.gov.gchq.palisade.Generated;
+import uk.gov.gchq.palisade.example.hrdatagenerator.types.Employee;
+import uk.gov.gchq.palisade.example.util.ExampleFileUtil;
+import uk.gov.gchq.palisade.resource.Resource;
+import uk.gov.gchq.palisade.resource.impl.DirectoryResource;
+import uk.gov.gchq.palisade.resource.impl.FileResource;
+import uk.gov.gchq.palisade.service.PolicyConfiguration;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.StringJoiner;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.gchq.palisade.example.common.ExamplePolicies.getParent;
+
+@ConfigurationProperties(prefix = "population")
+public class ExamplePolicyConfiguration implements PolicyConfiguration {
+
+    private String resource;
+    private List<ExamplePolicyCacheWarmerFactory> policies = new ArrayList<>();
+
+    /**
+     * Constructor with 0 arguments for an example implementation
+     * of the {@link PolicyConfiguration} interface
+     */
+    public ExamplePolicyConfiguration() {
```

Comment:
does this want to instantiate the resource and policies fields with empty values?

Reply:
Instantiates fields with empty values.